how-tos/merge-and-staging-instance.md

```diff
@@ -0,0 +1,10 @@
+# Staging and merging instance
+
+There is an AWS instance to run staging and merging instances so that we are fewer hops away from the hg repos.  
+
+To access it, you can either 
+1. Create an new instance with the AMI named "Mozilla release staging/merge machine".  I used a M3.large instance with 32 gb space added.
```

Comment:
formatting tiny nit: can you please enclose the machine type within code-quotes like ```this```? I find it easier to spot and copy-paste info if it's highlighted.